### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by SuveenE**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -232,24 +232,25 @@ tailored to their individual needs, helping them achieve their fitness goals ami
...
-| Adding new exercise to database and log it:           | `add e/<EXERCISE> c/<CALORIES_BURNT_PER_MIN(kCal)> d/<DURATION_PER_MIN>`   |
+| Log meal| `log m/<MEAL_NAME> s/<NUMBER_OF_SERVINGS>`
+| Log water intake | `log w/<AMOUNT_OF_WATER>`
+| Log exercises| `log e/<EXERCISE_NAME> d/<DURATION>`
+| View daily goals| `show g`
+| View daily calorie intake| `show c`
+| View daily water intake| `show w`
+| View all logged entries| `list`
+| View all food items in the database| `foodData`
+| View all exercises in the database| `exerciseData`
```
Comment (alvinnnnnnnnnn) [inline docs/DeveloperGuide.md:243]: I think its FoodData and ExerciseData

File: docs/UserGuide.md [hunk 2]
```diff
@@ -17,177 +17,271 @@ follow these simple steps:
...
+- [Quick Start](#quick-start)
+- [Features](#features)
+    - [Log meal: `log`](#log-meal--log)
+    - [Log water intake : `log`](#log-water-intake--log)
+    - [Log exercises: `log`](#log-exercises-log)
+    - [View daily goals: `show`](#view-daily-goals-show)
+    - [View daily calorie intake: `show`](#view-daily-calorie-intake-show)
+    - [View daily water intake: `show`](#view-daily-water-intake-show)
+    - [View all logged entries: `list`](#view-all-logged-entries-list)
+    - [View all food items in the database: `foodData`](#view-all-food-items-in-the-database-fooddata)
```
Comment (alvinnnnnnnnnn) [inline docs/UserGuide.md:31]: same as above

File: docs/UserGuide.md [hunk 3]
```diff
@@ -210,23 +304,24 @@ important to be careful when using the clear command.
...
+
+1. Log meal: `log m/<MEAL_NAME> s/<NUMBER_OF_SERVINGS>`    
+2. Log water intake : `log w/<AMOUNT_OF_WATER>`    
+3. Log exercises: `log e/<EXERCISE_NAME> d/<DURATION>`     
+4. View daily goals: `show g`  
+5. View daily calorie intake: `show c`     
+6. View daily water intake: `show w`      
+7. View all logged entries: `list`     
+8. View all food items in the database: `foodData`     
+9. View all exercises in the database: `exerciseData`
```
Comment (alvinnnnnnnnnn) [inline docs/UserGuide.md:317]: same as above

File: docs/UserGuide.md [hunk 4]
```diff
@@ -17,177 +17,271 @@ follow these simple steps:
...
+Expected output:
+---
 
-* The `<CALORIE_GOAL>` can be calorie goal for the day.
-* The `<WATER_GOAL>` can be water goal for the day
+### View all food items in the database: `foodData`
+Displays all the food items in the application database.
 
-Example of usage:
+Sample input: `foodData`
```
Comment (alvinnnnnnnnnn) [inline docs/UserGuide.md:130]: same as above

**Review by alvinnnnnnnnnn — CHANGES_REQUESTED** [review]
just the caps in FoodData and ExerciseData

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (SuveenE) [reply docs/DeveloperGuide.md:243]: It works for foodData with simple also. 

I think it's better to keep the first letter simple as all other commands start with a simple letter.

**Comment by alvinnnnnnnnnn** [comment]
the rest looks good to me!

**Comment by alvinnnnnnnnnn** [comment]
looks good to me!

File: src/main/java/command/DeleteTaskCommand.java [hunk 5]
```diff
@@ -20,6 +20,10 @@ public DeleteTaskCommand(String inputTrimmed) {
                 indexParts = parts[1].trim().split("i/");
                 this.index = Integer.parseInt(indexParts[1].trim());
                 this.description = indexParts[0].trim();
+                if(index <= 0){
+                    CommandUi.printInvalidItemIndexMessage();
+                    this.errorRaised = true;
+                }
```
Comment (alvinnnnnnnnnn) [inline src/main/java/command/DeleteTaskCommand.java:26]: looks good to me!

**Review by alvinnnnnnnnnn — APPROVED** [review]

**Pull request merged**